blocks/multi-panel-carousel/multi-panel-carousel.js

```diff
@@ -560,7 +561,9 @@ async function generateDropDowns(block, type, marginActions) {
             count += 1;
           });
         }
-        addingDynamicDropDowns(block, type, restructuredData, marginActions);
+        if (restructuredData[dropDownDetails.dropdownType].length > 1) {
```

Comment:
Do we need any null safety here?

Reply:
@sdmcraft there will always be some value in this https://github.com/aemsites/icicidirect/pull/255/files#diff-df1b2d463c6833ad8072b28c5a30afb20ba6675ffa4fad33cc4a1e287b7cfe1fL546. So no need for null check